jetson/odrive_bridge/src/__main__.py

```diff
@@ -39,7 +38,6 @@ def main():
     start_time = t.clock()
 
     legal_controller = int(sys.argv[1])
-    legal_axis = sys.argv[2]

```

Comment:
did we not choose specific axes when booting odrive_bridge before?

Reply:
at one point i think we did -- we don't anymore its smth that's now deprecated